oat-sa/tao-item-runner-qti-fe
# Fix/NSA-106/Update ckeditor

**Related to task:** https://oat-sa.atlassian.net/browse/NSA-106
**Description:** The "change" event is implemented in [the latest version of CKEditor ](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_savedata.html#observing-changes-in-ckeditor), so extra plugin "onchange" is removed

**Requires:** 
- [ ] Should be released together with [@oat-sa/tao-core-shared-libs](https://github.com/oat-sa/tao-core-shared-libs-fe/pull/3)

**How to test:** 
- Install [@oat-sa/tao-core-shared-libs](https://github.com/oat-sa/tao-core-shared-libs-fe/pull/3)
- Create a new item with "Extended Text Interaction"
- On windows OS to copy/past content from Word Document (you can find file in the ticket) should work
- Please check if CKEditor is working correctly on other places: metadata, test runner.
